app/models/taxon.rb

```diff
@@ -19,7 +19,7 @@ class Taxon
 
   validates_presence_of :title, :description, :internal_name, :path_prefix
   validates :path_prefix, inclusion: { in: Theme.taxon_path_prefixes }
-  validates :path_slug, allow_blank: true, format: { with: %r{\A/[a-zA-Z0-9\-]+\z} }
+  validates :path_slug, allow_blank: true, format: { with: %r{\A/[a-zA-Z0-9\-\/]+\z} }
```

Comment:
This could really do with being in it's own commit, so there's a historical record of why this change was made.